File: advanced/Conditional_Iterative/BMI/bmi.py

def bmiCalculator():
    height = float(input("Enter the height in meter: "))
    weight = float(input("Enter the weight in kilo: "))
    bmi = weight / (height**2)
    res = ""
    if (bmi >= 0 and bmi <= 18.5):
        res = "Underweight"
    elif (bmi > 18.5 and bmi <= 25):
        res = "Normal weight"
    elif (bmi > 25 and bmi <= 30):
        res = "Pre-obesity"
    elif (bmi > 30 and bmi <= 35):
        res = "Obesity class I"
    elif (bmi > 35 and bmi <= 40):
        res = "Obesity class II"
    elif (bmi > 40):
        res = "Obesity class III"
    else:
        res = "Enter a positive number"
    return "The person with height: {:.2f} and weight: {:.2f} is {}".format(height, weight, res)

print(bmiCalculator())

File: advanced/Conditional_Iterative/BMI/bmi.php
<?php
function bmiCalculator(){
    $weight = readline("Enter your weight in kilograms: ");
    $height = readline("Enter your height in meter: ");

    $bmi = $weight/($height**2);
    if ($bmi<0){
        $res = "weight and height must be positive numbers!";
    } elseif ($bmi>= 0 && $bmi<=18.5){
        $res = "Underweight";
    } elseif ($bmi> 18.5 && $bmi<=25){
        $res= "Normal weight";
    } elseif ($bmi> 25 && $bmi<=30){
        $res = "Pre-obesits";
    } elseif ($bmi> 30 && $bmi<=35){
        $res = "Obesity class I";
    } elseif ($bmi> 35 && $bmi<=40){
        $res = "Obesity class II";
    } elseif ($bmi> 40){
        $res = "Obesity class III";
    }

    if ($weight>= 0 && $height>= 0){
        return ("${res}, you Body mass index is: $bmi");
    } else {return $res;};
};

print_r(bmiCalculator())
?>